app/modules/aws/identity_center.py

```diff
@@ -0,0 +1,316 @@
+"""Module to sync the AWS Identity Center with the Google Workspace."""
+from logging import getLogger
+from integrations.aws import identity_store
+from modules.provisioning import users, groups
+from utils import filters
+
+
+logger = getLogger(__name__)
+DRY_RUN = True
```

Comment:
So the DRY_RUN will always be true - is this by design that you want it right now and then enable it later? Just wondering